submissions/Neskazhuk/popup/index.html

```diff
@@ -0,0 +1,138 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Popup</title>
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+<link href="https://fonts.googleapis.com/css2?family=Montserrat:wght@400;600&display=swap" rel="stylesheet">
+    <link href="styles.css" rel="stylesheet">
+</head>
+
+<body>
+    <header class="header">
+        <nav class="nav">
+            <ul class="nav-list">
+                <li class="nav-item"><a class="link" href="#">Hello</a></li>
+                <li class="nav-item"><a class="link" href="#">World</a></li>
+                <li class="nav-item popup">
+                    <input id="popup-input-id" class="popup-input" type="checkbox" tabindex="0">
+                    <label for="popup-input-id" class="popup-btn">
+                        <img class="popup-icon" src="./icons/popup-button.png" alt="">
+                    </label>
+                    <div class="popup-container">
+                       <input id="more-input" class="more-input" type="checkbox"> 
```

Comment:
When popup is opened focus should go to the first interactive element inside(usually it's close button) in our case when the user opened popup and presses `Tab` key focus must go to the first link in popup.

Reply:
Fixed the order using tabindex for all items. This is the only way I found, without changing the whole structure of the document. Hope, you'll accept it)